web/app/css/breadcrumb-header.css

```diff
@@ -12,6 +12,7 @@
      color: #404544;
      font-size: 22px;
      font-weight: 500;
```

Comment:
Since you're here, do you mind changing this to `font-weight: var(--font-weight-bold);`? I think it's a little more header-y and uses our default weights.